src/subscribe.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: enterprise
+title: Tilt Dev News
+---
+
+<h2 class="Enterprise-heroTitle">Tilt Dev News</h2>
```

Comment:
```suggestion
<h2 class="Enterprise-heroTitle">Tilt News</h2>
```

Is our brand 'Tilt' or 'Tilt Dev'? 🤔

Reply:
lately i tend to use "Tilt Dev", simply because it's easier to google

Reply:
Sounds good